cypress/integration/addNewClient/client.js

```diff
@@ -67,31 +78,37 @@ describe("Test for adding new client ", () => {
       const editemail = faker.internet.email();
       const mail = faker.internet.email();
 
-      client.typeEmail("1", mail).clickSave("Save").wait(3000);
+      client.selectAssignee();
+
+      cy.wait(2000);
+
+      client.typeEmail("1", mail).clickOnSaveButton();
```

Comment:
typeEmail might not be needed here since it is already typed in beforeEach function